receiver/awsxrayreceiver/internal/translator/aws.go

```diff
@@ -25,7 +25,14 @@ func addAWSToResource(aws *awsxray.AWSData, attrs pcommon.Map) {
 	addString(aws.AccountID, conventions.AttributeCloudAccountID, attrs)
 
 	// based on https://docs.aws.amazon.com/xray/latest/devguide/xray-api-segmentdocuments.html#api-segmentdocuments-aws
-	// it's possible to have all ec2, ecs and beanstalk fields at the same time.
+	// it's possible to have all cloudwatch_logs, ec2, ecs and beanstalk fields at the same time.
+	if cwl := aws.CWLogs; cwl != nil {
```

Comment:
Is there a reason `cwl` is necessary here (and we can't directly refer to `aws.CWLogs`)?

Reply:
Just trying to follow the format of the others in the file. I don't think it's necessary either.